packages/gatsby-source-shopify/src/update-cache.ts

```diff
@@ -73,39 +74,63 @@ export async function updateCache(
   pluginOptions: IShopifyPluginOptions,
   lastBuildTime: Date
 ): Promise<void> {
-  const { typePrefix } = pluginOptions
-
-  const nodeMap: IShopifyNodeMap = Object.keys(shopifyTypes)
-    .map(type => gatsbyApi.getNodesByType(`${typePrefix}Shopify${type}`))
-    .reduce((acc, value) => acc.concat(value), [])
-    .reduce((acc, value) => {
-      return { ...acc, [value.id]: value }
-    }, {})
-
+  let timer = gatsbyApi.reporter.activityTimer(
+    `[gatsby-source-shopify debug] updateCache - fetch events since`
+  )
+  timer.start()
```

Comment:
those are just debug activities - they will be removed - they are just helpful when iterating on this, but are not really helpful to users